modules/ROOT/pages/fips-140-2-compliance-support.adoc

```diff
@@ -9,6 +9,9 @@ The Mule 4 Runtime can be configured to run in a FIPS 140-2 certified environmen
 * Have a certified cryptography module installed in your Java environment
 * Adjust Mule Runtime settings to run in FIPS security mode
 
+[NOTE]
+By default, Government Cloud is configured for FIPS 140-2, so you do not need to perform the following steps if you are using Government Cloud.
+
```

Comment:
So they don't need the "running" or "fine tuning" sections either?

Reply:
Not according to the tech reviewer.